test/neura/llvm_sub.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: mlir-neura-opt --assign-accelerator --lower-llvm-to-neura --insert-mov %s | FileCheck %s
```

Comment:
It is even better to add the new op into the interpreter/emulator: https://github.com/coredac/dataflow/blob/main/tools/neura-interpreter/neura-interpreter.cpp

Corresponding test: https://github.com/coredac/dataflow/blob/86aab3e66f46d1003fddb8b9c2f045ac2d12cf88/test/neura/interpreter/lower_and_interpreter.mlir#L25

The interpreter issue is here: https://github.com/coredac/dataflow/issues/5, no need to make it complete for all ops, but it might be great to add the per-op support when you add new one.

Reply:
Ops miss this part totally. let me get into it and get back.